source/blog/2018-03-08-update-kernel-arguments-on-atomic-host.html.md

```diff
@@ -179,8 +177,14 @@ The kernel arguments are:
 test=test console=ttyS0,115200n8 biosdevname=0 rd.lvm.lv=atomicos/root root=/dev/mapper/atomicos-root ostree=/ostree/boot.0/fedora-atomic/9a9b350be75846811cbb0b1fd7b3d42a49908ed1265bc59e292bb4a34674332c/0
 ```
 
+# Conclusion:
+
+Those are all the major functionalities for `rpm-ostree ex kargs`. Feel free to try this out, feedbacks are always welcome!
+To post feedback about this command, you can post a issue on [rpm-ostree repo](https://github.com/projectatomic/rpm-ostree/issues),
+or join #atomic on freenode to ask questions. Thank you for reading this document!
```

Comment:
Recommendation: Replace "Thank you for reading this document!" with "Thanks for reading!"